os_migrate/playbooks/deploy_conversion_hosts.yml

```diff
@@ -79,7 +79,21 @@
     ansible_become_user: root
     ansible_become: true
 
-- hosts: conversion_hosts
+- hosts: os_migrate_conv_src
+  any_errors_fatal: true
+  tasks:
+    - name: install the conversion host content
+      ansible.builtin.include_role:
+        name: os_migrate.os_migrate.conversion_host_content
+      when: os_migrate_conversion_host_content_install|default(true)|bool
+  vars:
+    ansible_become_user: root
+    ansible_become: true
+    os_migrate_conversion_host_pre_content_hook: "{{ os_migrate_src_conversion_host_pre_content_hook|default(false) }}"
+    os_migrate_conversion_host_post_content_hook: "{{ os_migrate_src_conversion_host_post_content_hook|default(false) }}"
+
+
+- hosts: os_migrate_conv_dst
```

Comment:
This is going to slow down the conversion host deployment, because until now, the installation of content went on in parallel, but this is going to make it go in series. I don't know of a better solution though, given that we have the variable name split. And the variable name split makes sense here. So +1